rediscluster/clu.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+"""
+redis cluster command line interface
+
+Copyright (c) 2017, Christian Matthaei
+"""
+
+from argparse import ArgumentParser
+from rediscluster import StrictRedisCluster
+from pprint import pprint
+import re
+import socket
+import readline
+
+
+def get_parser():
+    """Get argument parser -> ArgumentParser"""
+
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        '--node', '-n', help='hostname of any cluster node',
+        default='127.0.0.1'
+    )
+    parser.add_argument(
+        '--pw', '-a', help='pw to auth, same as for redis-cli'
```

Comment:
I do not like this short version argument `-a`. It makes no sense and i have not seen any other tool use `-a` for password. My suggestion is that you follow the standard `--user/-u` `--password/-p` but only use the `--port` argument.

One other solution would be that you bundle together IP and port into one argument like `--node 127.0.0.1:6379` and you parse that from the argument and you default to `6379` if not specified. Code should exists in the normal client that you can reuse or inside redis-py.

Reply:
Well, the original redis-cli is using the `-a` argument:) that's why I choosed it. Should I change it anyway?

Reply:
Good point, keep it as is to maintain compatibility with the old redis-cli client.